### PR TITLE
docs(api): Update api_status.md

### PR DESCRIPTION
**Pull request by dranikpg**
This is the API status of the master branch, right? (the website directly refers here and not to the latest release)

- WATCH/UNWATCH will be implemented as #277 will be merged soon
- I've looked up missing commands as by #91 to see what still needs to be implemented
- Some commands are already implemented but not listed

### CODE REVIEW

File: docs/api_status.md [hunk 1]
```diff
@@ -202,7 +202,23 @@ with respect to Memcached and Redis APIs.
   - [ ] PFMERGE
 
 ### API 3
+- [X] Generic Family
```
Comment (odedponcz) [inline docs/api_status.md:205]: Why is this checked?

File: docs/api_status.md [hunk 2]
```diff
@@ -231,4 +247,4 @@ Some commands were implemented as decorators along the way:
  - [X] ROLE (2.8) decorator as master.
  - [X] UNLINK (4.0) decorator for DEL command
```
Comment (odedponcz) [inline docs/api_status.md:248]: Please remove from here if you add at API 4

File: docs/api_status.md [hunk 3]
```diff
@@ -202,7 +202,23 @@ with respect to Memcached and Redis APIs.
...
+- [X] Server Family
+  - [ ] CLIENT REPLY
+  - [X] REPLCONF
+  - [ ] WAIT
+
 ### API 4
+
+- [X] Generic Family
+  - [X] UNLINK
+- [X] Server Faimily
```
Comment (odedponcz) [inline docs/api_status.md:218]: Faimily -> Family

Comment (odedponcz) [inline docs/api_status.md:218]: Why is this added if non are checked?

**Review by odedponcz — APPROVED** [review]
Please change before submitting

File: docs/api_status.md (hunk 3 above)
Comment (dranikpg) [reply docs/api_status.md:218]: To show commands that are not supported, like in HYPERLOGLOG or STREAM for API 6

**Review by romange — APPROVED** [review]
